lib/usd/ui/importDialog/USDImportDialogCmd.cpp

```diff
@@ -206,8 +218,24 @@ MStatus USDImportDialogCmd::doIt(const MArgList& args)
             // toggle the wait cursor to show that it's working.
             QApplication::setOverrideCursor(QCursor(Qt::WaitCursor));
 
-            std::unique_ptr<USDImportDialog> usdImportDialog(new USDImportDialog(
-                assetPath.asChar(), &importData, usdQtUtil, MQtUtil::mainWindow()));
+            QWidget*      parentWindow = MQtUtil::mainWindow();
+            const MString parentWindowName
+                = parseTextArg(MArgDatabase(syntax(), args), kParentWindowFlag, "");
```

Comment:
Do you really need to create a MArgDatabase here? Can't you just use the MArgParser variable (argData) from above?

Reply:
Oh I reused code from somewhere else without analyzing if it could be optimized.